Index: Bronze/src/augustW2/B2446.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/augustW2/B2446.java b/Study/src/bronze/B2446.java
rename from Bronze/src/augustW2/B2446.java
rename to Study/src/bronze/B2446.java
--- a/Bronze/src/augustW2/B2446.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B2446.java	(date 1718112396707)
@@ -1,4 +1,4 @@
-package augustW2;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/bronze/B15740.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/bronze/B15740.java b/Study/src/bronze/B15740.java
rename from Bronze/src/bronze/B15740.java
rename to Study/src/bronze/B15740.java
--- a/Bronze/src/bronze/B15740.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B15740.java	(date 1718112396687)
@@ -1,4 +1,4 @@
-package Bronze.src.bronze;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/bronze/B3046.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/bronze/B3046.java b/Study/src/bronze/B3046.java
rename from Bronze/src/bronze/B3046.java
rename to Study/src/bronze/B3046.java
--- a/Bronze/src/bronze/B3046.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B3046.java	(date 1718112396677)
@@ -1,4 +1,4 @@
-package Bronze.src.bronze;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/bronze/B11021.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/bronze/B11021.java b/Study/src/bronze/B11021.java
rename from Bronze/src/bronze/B11021.java
rename to Study/src/bronze/B11021.java
--- a/Bronze/src/bronze/B11021.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B11021.java	(date 1718112397053)
@@ -1,4 +1,4 @@
-package Bronze.src.bronze;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Solved/src/bronze/solved2752.java
===================================================================
diff --git a/Solved/src/bronze/solved2752.java b/Solved/src/bronze/solved2752.java
deleted file mode 100644
--- a/Solved/src/bronze/solved2752.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ /dev/null	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
@@ -1,28 +0,0 @@
-package bronze;
-
-import java.util.Scanner;
-
-public class solved2752 {
-
-	public static void main(String[] args) {
-		Scanner sc = new Scanner(System.in);
-		int a = sc.nextInt();
-		int b = sc.nextInt();
-		int c = sc.nextInt();
-		int k;
-
-		int[] arr = { a, b, c };
-
-		for (int i = 0; i < arr.length-1; i++) {
-			
-			if (arr[i] > arr[i + 1]) {
-				k = arr[i];
-				arr[i] = arr[i + 1];
-				arr[i + 1] = k;
-			}
-		}
-		for(int i = 0; i<arr.length;i++) {
-			System.out.print("i:"+i);
-		}
-	}
-}
Index: Solved/.project
===================================================================
diff --git a/Solved/.project b/Solved/.project
deleted file mode 100644
--- a/Solved/.project	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ /dev/null	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
@@ -1,17 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<projectDescription>
-	<name>Solved</name>
-	<comment></comment>
-	<projects>
-	</projects>
-	<buildSpec>
-		<buildCommand>
-			<name>org.eclipse.jdt.core.javabuilder</name>
-			<arguments>
-			</arguments>
-		</buildCommand>
-	</buildSpec>
-	<natures>
-		<nature>org.eclipse.jdt.core.javanature</nature>
-	</natures>
-</projectDescription>
Index: Solved/src/module-info.java
===================================================================
diff --git a/Solved/src/module-info.java b/Solved/src/module-info.java
deleted file mode 100644
--- a/Solved/src/module-info.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ /dev/null	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
@@ -1,9 +0,0 @@
-/**
- * 
- */
-/**
- * @author iseoyeon
- *
- */
-module Bronze {
-}
\ No newline at end of file
Index: Solved/.classpath
===================================================================
diff --git a/Solved/.classpath b/Solved/.classpath
deleted file mode 100644
--- a/Solved/.classpath	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ /dev/null	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<classpath>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-17">
-		<attributes>
-			<attribute name="module" value="true"/>
-		</attributes>
-	</classpathentry>
-	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="output" path="bin"/>
-</classpath>
Index: Bronze/src/augustW5/B2446.java
===================================================================
diff --git a/Bronze/src/augustW5/B2446.java b/Bronze/src/augustW5/B2446.java
deleted file mode 100644
--- a/Bronze/src/augustW5/B2446.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ /dev/null	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
@@ -1,18 +0,0 @@
-package augustW5;
-
-import java.util.Scanner;
-
-public class B2446 {
-
-	public static void main(String[] args) {
-		// TODO Auto-generated method stub
-
-		Scanner sc = new Scanner(System.in);
-		int N = sc.nextInt();
-				
-//		for(int i=0; i < N;i++) {
-//			for()
-//		}
-	}
-
-}
Index: Silver/.classpath
===================================================================
diff --git a/Silver/.classpath b/Silver/.classpath
deleted file mode 100644
--- a/Silver/.classpath	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ /dev/null	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<classpath>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-17">
-		<attributes>
-			<attribute name="module" value="true"/>
-		</attributes>
-	</classpathentry>
-	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="output" path="bin"/>
-</classpath>
Index: Silver/src/module-info.java
===================================================================
diff --git a/Silver/src/module-info.java b/Silver/src/module-info.java
deleted file mode 100644
--- a/Silver/src/module-info.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ /dev/null	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
@@ -1,9 +0,0 @@
-/**
- * 
- */
-/**
- * @author iseoyeon
- *
- */
-module Silver {
-}
\ No newline at end of file
Index: Silver/.project
===================================================================
diff --git a/Silver/.project b/Silver/.project
deleted file mode 100644
--- a/Silver/.project	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ /dev/null	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
@@ -1,17 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<projectDescription>
-	<name>Silver</name>
-	<comment></comment>
-	<projects>
-	</projects>
-	<buildSpec>
-		<buildCommand>
-			<name>org.eclipse.jdt.core.javabuilder</name>
-			<arguments>
-			</arguments>
-		</buildCommand>
-	</buildSpec>
-	<natures>
-		<nature>org.eclipse.jdt.core.javanature</nature>
-	</natures>
-</projectDescription>
Index: Bronze/src/julyW5/B2845.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/julyW5/B2845.java b/Study/src/bronze/B2845.java
rename from Bronze/src/julyW5/B2845.java
rename to Study/src/bronze/B2845.java
--- a/Bronze/src/julyW5/B2845.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B2845.java	(date 1718112396780)
@@ -1,4 +1,4 @@
-package julyW5;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/julyW5/B2525.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/julyW5/B2525.java b/Study/src/bronze/B2525.java
rename from Bronze/src/julyW5/B2525.java
rename to Study/src/bronze/B2525.java
--- a/Bronze/src/julyW5/B2525.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B2525.java	(date 1718112397041)
@@ -1,4 +1,4 @@
-package julyW5;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/julyW5/B2588.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/julyW5/B2588.java b/Study/src/bronze/B2588.java
rename from Bronze/src/julyW5/B2588.java
rename to Study/src/bronze/B2588.java
--- a/Bronze/src/julyW5/B2588.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B2588.java	(date 1718112396726)
@@ -1,4 +1,4 @@
-package julyW5;
+package Study.src.bronze;
 
 import java.util.Arrays;
 import java.util.Scanner;
Index: Bronze/src/julyW5/B1008.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/julyW5/B1008.java b/Study/src/bronze/B1008.java
rename from Bronze/src/julyW5/B1008.java
rename to Study/src/bronze/B1008.java
--- a/Bronze/src/julyW5/B1008.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B1008.java	(date 1718112397018)
@@ -1,4 +1,4 @@
-package julyW5;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/julyW5/B2857.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/julyW5/B2857.java b/Study/src/bronze/B2857.java
rename from Bronze/src/julyW5/B2857.java
rename to Study/src/bronze/B2857.java
--- a/Bronze/src/julyW5/B2857.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B2857.java	(date 1718112396717)
@@ -1,4 +1,4 @@
-package julyW5;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/julyW5/B1271.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/julyW5/B1271.java b/Study/src/bronze/B1271.java
rename from Bronze/src/julyW5/B1271.java
rename to Study/src/bronze/B1271.java
--- a/Bronze/src/julyW5/B1271.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B1271.java	(date 1718112396952)
@@ -1,4 +1,4 @@
-package julyW5;
+package Study.src.bronze;
 
 import java.math.BigInteger;
 import java.util.Scanner;
Index: Bronze/src/julyW5/B1000.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/julyW5/B1000.java b/Study/src/bronze/B1000.java
rename from Bronze/src/julyW5/B1000.java
rename to Study/src/bronze/B1000.java
--- a/Bronze/src/julyW5/B1000.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B1000.java	(date 1718112396760)
@@ -1,4 +1,4 @@
-package julyW5;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/julyW5/B10718.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/julyW5/B10718.java b/Study/src/bronze/B10718.java
rename from Bronze/src/julyW5/B10718.java
rename to Study/src/bronze/B10718.java
--- a/Bronze/src/julyW5/B10718.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B10718.java	(date 1718112396988)
@@ -1,4 +1,4 @@
-package julyW5;
+package Study.src.bronze;
 
 public class B10718 {
 
Index: Bronze/src/julyW5/B1330.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/julyW5/B1330.java b/Study/src/bronze/B1330.java
rename from Bronze/src/julyW5/B1330.java
rename to Study/src/bronze/B1330.java
--- a/Bronze/src/julyW5/B1330.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B1330.java	(date 1718112396995)
@@ -1,4 +1,4 @@
-package julyW5;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/julyW5/B2441.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/julyW5/B2441.java b/Study/src/bronze/B2441.java
rename from Bronze/src/julyW5/B2441.java
rename to Study/src/bronze/B2441.java
--- a/Bronze/src/julyW5/B2441.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B2441.java	(date 1718112396777)
@@ -1,4 +1,4 @@
-package julyW5;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/augustW1/B2439.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/augustW1/B2439.java b/Study/src/bronze/B2439.java
rename from Bronze/src/augustW1/B2439.java
rename to Study/src/bronze/B2439.java
--- a/Bronze/src/augustW1/B2439.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B2439.java	(date 1718112396751)
@@ -1,4 +1,4 @@
-package augustW1;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/augustW1/B2914.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/augustW1/B2914.java b/Study/src/bronze/B2914.java
rename from Bronze/src/augustW1/B2914.java
rename to Study/src/bronze/B2914.java
--- a/Bronze/src/augustW1/B2914.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B2914.java	(date 1718112397046)
@@ -1,4 +1,4 @@
-package augustW1;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/augustW1/B1152.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/augustW1/B1152.java b/Study/src/bronze/B1152.java
rename from Bronze/src/augustW1/B1152.java
rename to Study/src/bronze/B1152.java
--- a/Bronze/src/augustW1/B1152.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B1152.java	(date 1718112396958)
@@ -1,4 +1,4 @@
-package augustW1;
+package Study.src.bronze;
 
 import java.util.Scanner;
 import java.util.StringTokenizer;
Index: Bronze/src/augustW1/B2420.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/augustW1/B2420.java b/Study/src/bronze/B2420.java
rename from Bronze/src/augustW1/B2420.java
rename to Study/src/bronze/B2420.java
--- a/Bronze/src/augustW1/B2420.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B2420.java	(date 1718112397001)
@@ -1,4 +1,4 @@
-package augustW1;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/julyW5/B2438.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/julyW5/B2438.java b/Study/src/bronze/B2438.java
rename from Bronze/src/julyW5/B2438.java
rename to Study/src/bronze/B2438.java
--- a/Bronze/src/julyW5/B2438.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B2438.java	(date 1718112397059)
@@ -1,4 +1,4 @@
-package julyW5;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/julyW5/B1284.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/julyW5/B1284.java b/Study/src/bronze/B1284.java
rename from Bronze/src/julyW5/B1284.java
rename to Study/src/bronze/B1284.java
--- a/Bronze/src/julyW5/B1284.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B1284.java	(date 1718112397009)
@@ -1,4 +1,4 @@
-package julyW5;
+package Study.src.bronze;
 
 import java.util.Scanner;
 import java.util.stream.Stream;
Index: Bronze/src/augustW1/B11721.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/augustW1/B11721.java b/Study/src/bronze/B11721.java
rename from Bronze/src/augustW1/B11721.java
rename to Study/src/bronze/B11721.java
--- a/Bronze/src/augustW1/B11721.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B11721.java	(date 1718112396703)
@@ -1,4 +1,4 @@
-package augustW1;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/augustW1/B1550.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/augustW1/B1550.java b/Study/src/bronze/B1550.java
rename from Bronze/src/augustW1/B1550.java
rename to Study/src/bronze/B1550.java
--- a/Bronze/src/augustW1/B1550.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B1550.java	(date 1718112396692)
@@ -1,4 +1,4 @@
-package augustW1;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/augustW2/B8393.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/augustW2/B8393.java b/Study/src/bronze/B8393.java
rename from Bronze/src/augustW2/B8393.java
rename to Study/src/bronze/B8393.java
--- a/Bronze/src/augustW2/B8393.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B8393.java	(date 1718112396682)
@@ -1,4 +1,4 @@
-package augustW2;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/augustW2/B2558.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/augustW2/B2558.java b/Study/src/bronze/B2558.java
rename from Bronze/src/augustW2/B2558.java
rename to Study/src/bronze/B2558.java
--- a/Bronze/src/augustW2/B2558.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B2558.java	(date 1718112396784)
@@ -1,4 +1,4 @@
-package augustW2;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/augustW5/B10950.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/augustW5/B10950.java b/Study/src/bronze/B10950.java
rename from Bronze/src/augustW5/B10950.java
rename to Study/src/bronze/B10950.java
--- a/Bronze/src/augustW5/B10950.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B10950.java	(date 1718112396963)
@@ -1,4 +1,4 @@
-package augustW5;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/augustW2/B6321.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/augustW2/B6321.java b/Study/src/bronze/B6321.java
rename from Bronze/src/augustW2/B6321.java
rename to Study/src/bronze/B6321.java
--- a/Bronze/src/augustW2/B6321.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B6321.java	(date 1718112396973)
@@ -1,4 +1,4 @@
-package augustW2;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/augustW2/B2741.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/augustW2/B2741.java b/Study/src/bronze/B2741.java
rename from Bronze/src/augustW2/B2741.java
rename to Study/src/bronze/B2741.java
--- a/Bronze/src/augustW2/B2741.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B2741.java	(date 1718112397028)
@@ -1,4 +1,4 @@
-package augustW2;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/augustW2/B2442.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/augustW2/B2442.java b/Study/src/bronze/B2442.java
rename from Bronze/src/augustW2/B2442.java
rename to Study/src/bronze/B2442.java
--- a/Bronze/src/augustW2/B2442.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B2442.java	(date 1718112396738)
@@ -1,4 +1,4 @@
-package augustW2;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/augustW2/B2885.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/augustW2/B2885.java b/Study/src/bronze/B2885.java
rename from Bronze/src/augustW2/B2885.java
rename to Study/src/bronze/B2885.java
--- a/Bronze/src/augustW2/B2885.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B2885.java	(date 1718112397035)
@@ -1,4 +1,4 @@
-package augustW2;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/augustW5/B15552.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/augustW5/B15552.java b/Study/src/bronze/B15552.java
rename from Bronze/src/augustW5/B15552.java
rename to Study/src/bronze/B15552.java
--- a/Bronze/src/augustW5/B15552.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B15552.java	(date 1718112396968)
@@ -1,4 +1,4 @@
-package augustW5;
+package Study.src.bronze;
 
 import java.io.BufferedReader;
 import java.io.BufferedWriter;
Index: Bronze/src/augustW5/B10869.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/augustW5/B10869.java b/Study/src/bronze/B10869.java
rename from Bronze/src/augustW5/B10869.java
rename to Study/src/bronze/B10869.java
--- a/Bronze/src/augustW5/B10869.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B10869.java	(date 1718112396980)
@@ -1,4 +1,4 @@
-package augustW5;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
Index: Bronze/src/augustW5/B10951.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Bronze/src/augustW5/B10951.java b/Study/src/bronze/B10951.java
rename from Bronze/src/augustW5/B10951.java
rename to Study/src/bronze/B10951.java
--- a/Bronze/src/augustW5/B10951.java	(revision 0c1a0ed890f5888c2704703ea66e1ff5db8f768c)
+++ b/Study/src/bronze/B10951.java	(date 1718112396712)
@@ -1,4 +1,4 @@
-package augustW5;
+package Study.src.bronze;
 
 import java.util.Scanner;
 
diff --git a/Bronze/src/module-info.java b/Study/src/module-info.java
rename from Bronze/src/module-info.java
rename to Study/src/module-info.java
diff --git a/Bronze/src/bronze/B10039.java b/Study/src/bronze/B10039.java
rename from Bronze/src/bronze/B10039.java
rename to Study/src/bronze/B10039.java
diff --git a/Bronze/src/bronze/B10952.java b/Study/src/bronze/B10952.java
rename from Bronze/src/bronze/B10952.java
rename to Study/src/bronze/B10952.java
diff --git a/Bronze/src/bronze/B1264.java b/Study/src/bronze/B1264.java
rename from Bronze/src/bronze/B1264.java
rename to Study/src/bronze/B1264.java
diff --git a/Bronze/src/bronze/B6749.java b/Study/src/bronze/B6749.java
rename from Bronze/src/bronze/B6749.java
rename to Study/src/bronze/B6749.java
diff --git a/Bronze/src/bronze/B2744.java b/Study/src/bronze/B2744.java
rename from Bronze/src/bronze/B2744.java
rename to Study/src/bronze/B2744.java
diff --git a/Bronze/src/bronze/B7287.java b/Study/src/bronze/B7287.java
rename from Bronze/src/bronze/B7287.java
rename to Study/src/bronze/B7287.java
diff --git a/Bronze/src/bronze/B5596.java b/Study/src/bronze/B5596.java
rename from Bronze/src/bronze/B5596.java
rename to Study/src/bronze/B5596.java
diff --git a/Bronze/src/bronze/B4101.java b/Study/src/bronze/B4101.java
rename from Bronze/src/bronze/B4101.java
rename to Study/src/bronze/B4101.java
diff --git a/Bronze/src/bronze/B11720.java b/Study/src/bronze/B11720.java
rename from Bronze/src/bronze/B11720.java
rename to Study/src/bronze/B11720.java
diff --git a/Bronze/src/bronze/B10172.java b/Study/src/bronze/B10172.java
rename from Bronze/src/bronze/B10172.java
rename to Study/src/bronze/B10172.java
diff --git a/Bronze/src/bronze/B2440.java b/Study/src/bronze/B2440.java
rename from Bronze/src/bronze/B2440.java
rename to Study/src/bronze/B2440.java
diff --git a/Bronze/src/bronze/B10757.java b/Study/src/bronze/B10757.java
rename from Bronze/src/bronze/B10757.java
rename to Study/src/bronze/B10757.java
diff --git a/Bronze/src/bronze/B2743.java b/Study/src/bronze/B2743.java
rename from Bronze/src/bronze/B2743.java
rename to Study/src/bronze/B2743.java
diff --git a/Bronze/src/bronze/B5554.java b/Study/src/bronze/B5554.java
rename from Bronze/src/bronze/B5554.java
rename to Study/src/bronze/B5554.java
diff --git a/Bronze/src/bronze/B5717.java b/Study/src/bronze/B5717.java
rename from Bronze/src/bronze/B5717.java
rename to Study/src/bronze/B5717.java
diff --git a/Bronze/src/bronze/B10171.java b/Study/src/bronze/B10171.java
rename from Bronze/src/bronze/B10171.java
rename to Study/src/bronze/B10171.java
diff --git a/Bronze/.project b/Study/.project
rename from Bronze/.project
rename to Study/.project
diff --git a/Bronze/.classpath b/Study/.classpath
rename from Bronze/.classpath
rename to Study/.classpath
